web/src/styles/style.css

```
@@ -141,48 +141,80 @@ h2 {
 /* === ESTILOS PARA LOGIN CON EMOJIS (NUEVO) === */
 /* ============================================= */
 .login-container {
-  position: absolute;
+  position: fixed;
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
-  background: rgba(0, 0, 0, 0.7);
+
+  background: rgba(0, 0, 0, 0.55);
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)